pinc/ProjectSearchResults.inc

```diff
@@ -4,7 +4,7 @@ include_once($relPath.'ProjectTransition.inc');
 include_once($relPath.'forum_interface.inc');
 include_once($relPath.'wordcheck_engine.inc');
 include_once($relPath.'ProjectSearchResultsConfig.inc');
-include_once($relPath.'forum_interface.inc'); // get_topic_details() & get_url_to_view_topic()
+include_once($relPath.'forum_interface.inc'); // get_url_to_view_topic()
```

Comment:
This change is intentional as `get_topic_details()` is not used in this file so the comment is incorrect.